roles/pulp/README.md

```diff
@@ -94,6 +107,22 @@ Role Variables
   subscription-manager/katello.
   Also accepts a single string or empty string.
   Only affects RHEL7 (RHEL8 no longer has an optional repo.)
+* pulp_django_admin_paths: A list of possible filepaths to the `django-admin`
```

Comment:
Can these rpm vars be grouped into their own area in the docs?

Reply:
This one isn't specific to RPM, just currently only needed by it.

That said, I was thinking of doing that. I'll look at the docs, and propose something.

I would also love to rework all the existing pip specific variables to begin with `pulp_pip`. Possibly even separate pulp_install_plugins into 2 different data structures for the 2 installer modes (`pulp_install_source`.)